frame/assets/src/functions.rs

```diff
@@ -300,6 +301,7 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		ensure!(!Account::<T, I>::contains_key(id, &who), Error::<T, I>::AlreadyExists);
 		let deposit = T::AssetAccountDeposit::get();
 		let mut details = Asset::<T, I>::get(&id).ok_or(Error::<T, I>::Unknown)?;
+		ensure!(details.status == AssetStatus::Live, Error::<T, I>::AssetNotLive);
```

Comment:
It the `is_frozen` depends on the asset status, then this is superfluous.

Reply:
I don't think they depend on each other, since they have different meanings. But in a future PR, we might also make is_frozen an asset status.

Reply:
it's probably simpler to do it with current changes, the migration already in place.
If not, let's have an issue.